source/docs/power-and-electronics/motor-guide/motor-current.rst

```diff
@@ -0,0 +1,34 @@
+Motor Power
+===========
+
+Just like any electrical device, motors draw current and consume a certain amount of power to function. However, motors will convert some percentage of the power that they draw into output power in the form of the shaft spinning. Thinking of motors in terms of energy instead of speed and torque can make it easier to do math with that motor and provide tools for choosing the optimal gear ratio for an application.
+
+What is power?
+--------------
+
+Fundamentally, power is defined as the amount of energy transferred in a given time period. Practically, this means power is how much "energy" you can get out of the motor. **The output power of a motor varies depending on how much load is on it, but does not change as the gear ratio of the motor changes**. What this means is that a free-spinning 1:1 motor will output the same amount of power as a free spinning 100:1 motor, but the actual speed and torque will be different. Output power is proportional to speed times torque, so as speed and torque are changed with a gearbox, the output power must remain the same.
+
+Another piece of information, although not always as useful, is the *input* power of the motor. The input power of the motor is how many watts of power the motor consumes, which is equal to the voltage sent to the motor times the current consumed by the motor. **Motors are not 100% efficient, so to get a certain output power you often need double or even triple the input power**. For example, a standard FTC motor can consume up to 65 watts just to produce 29 watts of output power.
+
+Peak power and motor curves
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to figure out how much power your motor will be consuming or outputting, its helpful to reference a motor curve. These are data sheets that plot the output speed, output torque, output power, and efficiency all on one graph. Every FTC legal motor has been tested by VEX Robotics, and their data was made available at `motors.vex.com <https://motors.vex.com>`_ (note: you will need to find your specific motor here, generally it will be a "REV Robotics HD Hex Motor" for REV motors, or "Modern Robotics 12VDC Motor" for gobilda motors). It is safe to assume other motors (such as servos) follow similar curves, although with different speed, torques, and power outputs. An example motor curve for a Modern Robotics 12v motor is shown below.
```

Comment:
```suggestion
In order to figure out how much power your motor will be consuming or outputting, its helpful to reference a motor curve. These are data sheets that plot the motor's output speed, output torque, output power, and efficiency all on one graph. Every FTC legal motor except the Core Hex has been tested by VEX Robotics, and their data were made available at `motors.vex.com <https://motors.vex.com>`_. There, you will need to find your specific motor, generally it will be a "REV Robotics HD Hex Motor" for REV motors, or "Modern Robotics 12VDC Motor" for goBILDA motors. It is safe to assume other motors (such as servos) follow similarly-shaped curves, although with different speed, torque, and power outputs. An example motor curve for a Modern Robotics 12 V motor is shown below.
```